packages/api-explorer/src/index.jsx

```diff
@@ -82,25 +83,27 @@ class ApiExplorer extends React.Component {
             <VariablesContext.Provider value={this.props.variables}>
               <OauthContext.Provider value={this.props.oauth}>
                 <GlossaryTermsContext.Provider value={this.props.glossaryTerms}>
-                  <SelectedAppContext.Provider value={this.state.selectedApp}>
-                    <Doc
-                      key={doc._id}
-                      doc={doc}
-                      oas={this.getOas(doc)}
-                      setLanguage={this.setLanguage}
-                      flags={this.props.flags}
-                      user={this.props.variables.user}
-                      Logs={this.props.Logs}
-                      baseUrl={this.props.baseUrl.replace(/\/$/, '')}
-                      appearance={this.props.appearance}
-                      language={this.state.language}
-                      oauth={this.props.oauth}
-                      suggestedEdits={this.props.suggestedEdits}
-                      tryItMetrics={this.props.tryItMetrics}
-                      auth={this.state.auth}
-                      onAuthChange={this.onAuthChange}
-                    />
-                  </SelectedAppContext.Provider>
+                  <BaseUrlContext.Provider value={this.props.baseUrl.replace(/\/$/, '')}>
+                    <SelectedAppContext.Provider value={this.state.selectedApp}>
+                      <Doc
+                        key={doc._id}
+                        doc={doc}
+                        oas={this.getOas(doc)}
+                        setLanguage={this.setLanguage}
+                        flags={this.props.flags}
+                        user={this.props.variables.user}
+                        Logs={this.props.Logs}
+                        baseUrl={this.props.baseUrl.replace(/\/$/, '')}
```

Comment:
Does this still need to be here?

Reply:
Yes I believe so - we need to pass down the baseUrl prop into two files for [docs](https://github.com/readmeio/api-explorer/blob/fix/baseUrl-in-anchors/packages/api-explorer/src/Doc.jsx) and [logs](https://github.com/readmeio/api-explorer/blob/fix/baseUrl-in-anchors/packages/api-logs/index.jsx).